wandb/apis/public/artifacts.py

```diff
@@ -528,21 +528,236 @@ def description(self):
         """A description of the artifact collection."""
         return self._description
 
+    @description.setter
+    def description(self, description: Optional[str]) -> None:
+        """Set the description of the artifact collection.
+
+        Arguments:
+            description: (str) Free text that offers a description of the artifact collection.
+        """
```

Comment:
Conventionally, a docstring is only added to the getter. IDEs ignore setter docstrings. You can delete the setter docstring.